schema/common/defaults/configuration/roles-mapping.yml

```diff
@@ -1,58 +1,9 @@
-kind: configuration/feature-mapping
+---
+kind: configuration/roles-mapping
 title: "Feature mapping to roles"
 name: default
 specification:
-  available_roles:
-    - name: repository
-      enabled: true
-    - name: firewall
-      enabled: true
-    - name: image-registry
-      enabled: true
-    - name: kubernetes-master
-      enabled: true
-    - name: kubernetes-node
-      enabled: true
-    - name: helm
-      enabled: true
-    - name: logging
-      enabled: true
-    - name: opendistro-for-elasticsearch
-      enabled: true
-    - name: elasticsearch-curator
-      enabled: true
-    - name: kibana
-      enabled: true
-    - name: filebeat
-      enabled: true
-    - name: prometheus
-      enabled: true
-    - name: grafana
-      enabled: true
-    - name: node-exporter
-      enabled: true
-    - name: jmx-exporter
-      enabled: true
-    - name: zookeeper
-      enabled: true
-    - name: kafka
-      enabled: true
-    - name: rabbitmq
-      enabled: true
-    - name: kafka-exporter
-      enabled: true
-    - name: postgresql
-      enabled: true
-    - name: postgres-exporter
-      enabled: true
-    - name: haproxy
-      enabled: true
-    - name: applications
-      enabled: true
-    - name: rook
-      enabled: true
-
-  roles_mapping:
+  roles:
```

Comment:
```suggestion
  roles_mapping:
```

Reply:
But `roles` name is not appropriate here since e.g. `custom` key is not a role but a mapping name. Also items of arrays are not roles, e.g. a role name is `image_registry` not `image-registry`.